crytic_compile/platform/abstract_platform.py

```diff
@@ -154,6 +176,19 @@ def is_dependency(self, path: str) -> bool:
         """
         return False
 
+    @staticmethod
```

Comment:
What if we dont make it abstract, to avoid having a code duplication for all platforms, to just return None?